src/app/login/login.component.ts

```
@@ -117,7 +121,11 @@ export class LoginComponent implements OnInit {
         (res: any) => {
           if (res.statusCode === 200) {
             if (res.data.previlegeObj && res.data.previlegeObj[0]) {
-              localStorage.setItem(
+              this.cookieService.set('Jwttoken', res.data.Jwttoken);
+              delete res.data.Jwttoken;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)